tests/test_cache_utils.py

```diff
@@ -455,3 +456,58 @@ def test_static_cache_extra_left_padding(self):
     @unittest.skip("TODO @gante static cache's does not support beam search yet")
     def test_static_cache_beam_search(self):
         pass
+
+    @require_torch_gpu
+    def test_offloaded_cache_equivalent_to_dynamic_cache(self):
+        """Tests that OffloadedCache produces the same result as the default DynamicCache"""
+        model_name = "microsoft/Phi-3-mini-4k-instruct"
+        tokenizer = AutoTokenizer.from_pretrained(model_name)
+        model = AutoModelForCausalLM.from_pretrained(
+            model_name, device_map="auto", torch_dtype=torch.float16, attn_implementation="flash_attention_2"
```

Comment:
```suggestion
            model_name, device_map="auto", torch_dtype=torch.float16
```
FA2 places additional requirements on the test. Given that it is not relevant for the test itself, we should remove it 🤗

Reply:
I have removed flash attention from the tests.